modules/040-control-plane-manager/images/control-plane-manager/control-plane-manager

```diff
@@ -319,18 +319,18 @@ function converge_component() {
 
   case $KUBERNETES_VERSION in
     1.19 | 1.20 | 1.21)
-      experimental_option="--experimental-patches"
+      experimental_option="--experimental-patches $ROOTFS_DIR/etc/kubernetes/deckhouse/kubeadm/patches/"
       ;;
     *)
-      experimental_option="--patches"
+      experimental_option=""
```

Comment:
Why did we remove experimental_option for k8s>=1.22 ? kubeadm 1.22 and 1.23 has option --patches.
I think use experimental_option is more clear way.